### ISSUE TITLE
Why were certain names excluded?

### ISSUE THREAD

**Issue by dscorbett** [adobe-type-tools/agl-aglfn#8]
glyphlist.txt contains many entries like bdotaccent and kadeva that are neither incorrect nor in the PUA. What was the reason not to include them in aglfn.txt?

**Comment by miguelsousa** [comment]
**glyphlist.txt** (AGL; 1997) was meant to provide the key to bridge the gap between pre- and post-Unicode eras. This `glyph_name-to-Unicode` mapping file is really only necessary when a system needs to handle a stream of glyphs originated from a PostScript Type 1 font; this format doesn't have a `cmap` table and its semantics depend on Adobe-defined glyph names and encodings.

**aglfn.txt** (AGLFN; 2002) was devised during Adobe's font library conversion from Type 1 to OpenType. It's a subset of AGL and it's essentially meant to be an informative guideline for type designers and font developers — because during the design and production phases it's a lot more convenient to use intuitive names such as `alpha` (from which we can easily tell what the glyph is supposed to look like) rather than something like `u03B1`.

Once Unicode became the _de facto_ standardized way of providing semantics to glyph shapes, there was not much value in carrying forward an old model based on specifying arbitrary Western names for each and every glyph.

From a font technology perspective, glyph names are not essential —the biggest exception being legacy printer workflows— and the OpenType format permits making valid font files that contain no glyph names at all. In an Unicode+OpenType world what's essential is that the font internally maps each Unicode value to a `glyph index` (not `glyph name`) that contains an appropriate visual representation of that particular character.

The bottom line is, glyph names are really only useful during the development of a font, and they do not need to abide to any particular standard. Glyph names can be whatever the designer/developer prefers. The only limitations will be the ones imposed by the tools used for designing and producing the font.

**Comment by miguelsousa** [comment]
Another way to think of it is: `bdotaccent` and `kadeva` didn't make it to the AGLFN because at the time when the list was devised none of the fonts in the Adobe font library contained those glyphs.

**Comment by dscorbett** [comment]
That was interesting and informative; thank you!
